app/src/main/res/values/strings.xml

```diff
@@ -295,6 +297,8 @@
     <string name="notification_poll_description">Notifications about polls that have ended</string>
     <string name="notification_subscription_name">New toots</string>
     <string name="notification_subscription_description">Notifications when somebody you\'re subscribed to published a new toot</string>
+    <string name="notification_sign_up_name">Sign ups</string>
```

Comment:
Do you know how Mastodon Web names this in their interface? I feel like we need to call it "Signups (for Admins)" or something to not confuse regular users. 
Or even better, is there a way to detect if the current user is admin? Then we could just hide this notification type completely if not.

Reply:
In the WebUI, it is called 'New sign-ups' and hidden for non-admin users. (https://github.com/mastodon/mastodon/pull/16953/files#diff-0b4b4ff412a8d0db0b4625619d3633b2cc295c4e7f21316e7941bb4728446d50R169)

![image](https://user-images.githubusercontent.com/27999567/155910198-83e11c15-cce5-4f51-8141-9cea1d8292f8.png)


WebUI takes `is_staff` property from `#initial-state` element but this is only available for WebUI.

![image](https://user-images.githubusercontent.com/27999567/155910846-4c6e230a-2deb-4cf4-9e17-fefa7373f49a.png)